java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/config/domain/BuildContext.java

```diff
@@ -49,6 +51,8 @@
 public class BuildContext {
 
   private static final String DOCKERFILE_NAME = "Dockerfile";
+  private static final List<String> EXISITING_DOCKER_DIRS =
+      ImmutableList.of("", "src/main/docker/");
```

Comment:
Small thing, but I think it's cleaner/less error-prone to just declare the full dockerfile path in this list, rather than build it dynamically later. Something like
```java
  private static final List<String> EXISITING_DOCKER_DIRS = ImmutableList.of(DOCKERFILE_NAME, "src/main/docker/" + DOCKERFILE_NAME);
```